resources/overrides/library/sap.uxap.d.ts

```diff
@@ -75,3 +75,14 @@ declare module "sap/uxap/ObjectPageSubSectionMode" {
 	*/
 	export default ObjectPageSubSectionMode;
 }
+
+declare module "sap/uxap/BlockBaseColumnLayout" {
+	import {BlockBaseColumnLayout} from "sap/uxap/library";
+
+	/**
+	 * Used by the <code>BlockBase</code> control to define how many columns should it be assigned by the <code>objectPageSubSection</code>. The allowed values can be auto (subsection assigned a number of columns based on the parent objectPageLayout subsectionLayout property), 1, 2, 3 or 4 (This may not be a valid value for some <code>subSectionLayout</code>, for example, asking for 3 columns in a 2 column layout would raise warnings).
```

Comment:
```suggestion
	 * Used by the <code>BlockBase</code> control to define how many columns it should be assigned by the <code>objectPageSubSection</code>. The allowed values can be "auto" (subsection assigned a number of columns based on the parent objectPageLayout subsectionLayout property), 1, 2, 3 or 4 (This may not be a valid value for some <code>subSectionLayout</code> values, for example, asking for 3 columns in a 2-column layout would raise warnings).
```